SONAR-netCDF4/docs/tableBeamGroup1.adoc

```diff
@@ -153,6 +178,13 @@
 |int active_position_sensor(ping_time) |MA |Indicate the index of the position sensor used at the time of the ping to compute the platform position.
 |:valid_min = "0" | |
 |:long_name = "Active position sensor index" | |
+| | |
+|float sound_speed_at_transducer(ping_time) |O |Sound speed at tx transducer depth at ping time
```

Comment:
So this applies to all transducers?

Reply:
To be honest we never saw a system with two tx antenna. 
I know for sure that EM2040 M has an option for that but I do not know if it acts like two couple TX/Rx ie like two sounders on each side of the boat or not. 
On the kongsberg datagram format there is only one entry called sound speed at transducer but without reference to any transducer index and so.
I'll try to figure out where are the sound velocity probe installed and how many there could be.

Reply:
After checks, there is usually only one sound velocity probe and only one input on sounder for those data. Therefore we will not set this value per tx_beam and since sound speed is also used for Rx I will remove any mentions to tx. 
If you agree with this I will keep the name like it is to be clear that this is not real surface sound speed but the sound speed at the depth of the transducer (which is quite different for ROV for example)

|float sound_speed_at_transducer(ping_time) |O |Sound speed at transducer depth at the time of the ping